fda/functional_data.py

```diff
@@ -516,21 +522,276 @@ def shift(self, shifts, *, restrict_domain=False, extrapolation=None,
         """
         pass
 
-    @abstractmethod
-    def plot(self, ax=None, derivative=0, **kwargs):
-        """Plot the FData object.
+    def set_figure_and_axes(self, nrows, ncols):
+        """Set figure and its axes.
 
         Args:
-            ax (axis object, optional): axis over with the graphs are plotted.
-                Defaults to matplotlib current axis.
-            **kwargs: keyword arguments to be passed to the
-                matplotlib.pyplot.plot function.
+            nrows(int, optional): designates the number of rows of the figure to
+                plot the different dimensions of the image. ncols must be also
+                be customized in the same call.
```

Comment:
Could you remove `ncols must be also  be customized in the same call.`? It do not know what I was thinking about when I added it!

Reply:
Done.